### DISCUSSION TITLE
Respond to webhooks with an API call using PTB?

### DISCUSSION THREAD

**Discussion by iBug** [python-telegram-bot/python-telegram-bot#3284] [Q&A] [answered]
The TG documentation says:

> If you're using webhooks, you can perform a request to the Bot API while sending an answer to the webhook.

Per the docs, one can create a full payload for the API they want to call, plus a `method` key to specify the API method which is normally the last part of the API URL.

Does PTB include any support for this feature, like "crafting an API call without sending it"?

Benefit: Reduced RTT for API responses where an active connection to TG servers might be slow, or otherwise save execution time (like AWS Lambda bills).

**Comment by Bibo-Joshi** [comment, marked as answer]
Please see #713
